docs/functional-tests.md

```diff
@@ -87,10 +87,10 @@
 ## Resource mapping validation
 | Test description | Implemented |
 | :---------------- | :---------: 
-| Import with missing network resource mapping should be blocked | &cross; |
-| Import with incorrect network mapping should be blocked | &cross; |
-| Import with incorrect storage mapping should be blocked | &cross; |
-| Import with incorrect disk mapping should be blocked | &cross; |
+| Import with missing network resource mapping should be blocked | &check; |
+| Import with incorrect network mapping should be blocked | &check; |
+| Import with incorrect storage mapping should be blocked | &check; |
```

Comment:
Should we maybe write here: Import with nonexistent storage class should be blocked? and same for others?

Reply:
Fixed.